dashboard/app/models/ability.rb

```diff
@@ -289,6 +291,12 @@ def initialize(user)
       end
     end
 
+    if user.persisted?
```

Comment:
I'm assuming that in the future we could grant this ability a bit more generically (any student in an AP CS A Section led by a verified teacher, or something similar).

Reply:
yep! We can put whatever logic we want here.